src/app/src/main/kotlin/org/vaccineimpact/api/app/app_start/route_config/ModellingGroupRouteConfig.kt

```diff
@@ -22,6 +22,9 @@ object ModellingGroupRouteConfig : RouteConfig
                     .json()
                     .post()
                     .secure(writePermissions),
+            Endpoint("$baseUrl/user/memberships/", controller, "getContextUserModellingGroups")
```

Comment:
Is this a reasonable route? I can't really use /modelling-groups/memberships as it will tend to confuse the routing (it would interpret 'memberships' as a :group-id for the other route.

Reply:
I think either `/users/:username/modelling-groups`, `/users/current-user/modelling-groups/` or `/user/modelling-groups`. Or we could just implement something like `/user/roles/` or even just `/user/` which would return metadata about the current user, and let the front-end pull out membership roles as needed...